packages/instantsearch.js/src/helpers/components/__tests__/Snippet.test.tsx

```diff
@@ -87,7 +87,7 @@ describe('Snippet', () => {
     <span
       class="ais-Snippet-separator"
     >
-      , 
+      ,
```

Comment:
does this not mean there's no space between the items anymore?

Reply:
I've been subject to this, it could be a side-effect of Prettier.

Reply:
It is, when we run `lint:fix` it removes white space ate the end of each line. I updated the snapshots.